.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [1.18.x, 1.19.x]
+        go-version: [1.20.x]
```

Comment:
Originally added the new version here, but then realized we just use Go to install tools for used by project checks. So running once here is okay.